oc_dms_mirror/rest_api/app/__init__.py

```
@@ -1,12 +1,14 @@
 from flask import Flask, Blueprint
 
-dms_mirror_bp = Blueprint("dms_mirror_bp", __name__)
 from .routes import *
```

Ruff (0.14.2)

`from .routes import *` used; unable to detect undefined names

(F403)